README.rst

```diff
@@ -74,10 +74,9 @@ You will need to install the PyPi GDAL package with the following options::
 
 We also rely on VIPS_ (version 8.2+) to do fast image processing.
 
-Under Debian or Ubuntu, run the following to install it::
-
-    $ sudo apt-get install libvips libvips-dev
+Under Debian or Ubuntu, run the following to install it without the GUI nip2::
 
+    $ sudo apt-get install --no-install-recommends libvips libvips-dev
```

Comment:
Huh, this is unusual, why no recommends?

Reply:
That's the "without the GUI nip2" bit. I seem to have have lost a comment when I fixup'd instead of squash =(
nip2 installs firefox and maybe some other big things that we don't need